src/collection-view.js

```diff
@@ -471,7 +534,7 @@ const CollectionView = Backbone.View.extend({
   // view to keep in sync with the collection.
   _updateIndices(view, increment, index) {
     if (!this.sort) {
-      return;
+      return this;
```

Comment:
Since `_updateIndices` is private we don't need to return `this`.

That consistency is just for the public API.